### DISCUSSION TITLE
article/b5af3ca5-8f24-4e4a-b75a-4591377e4764

### DISCUSSION THREAD

**Discussion by giscus** [siuze/NotionNext#7] [Announcements]
# article/b5af3ca5-8f24-4e4a-b75a-4591377e4764

包括植物在内的各种物产名称是方言中具有强烈地域特性的一部分。同一植物在相邻的村镇也可能有不同的的称呼。我在这篇笔记里主要记录一些我自己村内通行的，具有明确对应的植物名称，用以备忘。

https://notion.siuze.top/article/b5af3ca5-8f24-4e4a-b75a-4591377e4764

**Comment by Guanchishan** [comment]
看到了最后一行。你那邊是用gang55單字表示菅芒嗎？

**Reply by siuze** [reply]
gang55表示五节芒，叶子会割人的那种；五节芒花后的杆子拿来做的扫帚叫菅帚 guang55 jiu33。    
芒萁这种蕨类我们这里叫gang21 mung55，不知道本字和“菅芒”有无关系，发音很怪。   
还有一些植物名称记在纸上尚未搬上来，近期没空写，等闲了再整理整理。

**Reply by Guanchishan** [reply]
> gang55表示五节芒，叶子会割人的那种；五节芒花后的杆子拿来做的扫帚叫菅帚 guang55 jiu33。    
> 芒萁这种蕨类我们这里叫gang21 mung55，不知道本字和“菅芒”有无关系，发音很怪。   
> 还有一些植物名称记在纸上尚未搬上来，近期没空写，等闲了再整理整理。

原来是五节芒呀👍

gang21 mung55到底是什么字，也令人很在意呢

**Comment by Guanchishan** [comment]
真不错。期待！
